### PR TITLE
url nav fix

### PR DESCRIPTION
**Pull request by tristanhcole**
A type error in `OrganisationWrapper.jsx` and `authSagas.js` was causing the URL query string to be broken.

### CODE REVIEW

File: app/client/components/pages/transferAccountListPage.jsx [hunk 1]
```diff
@@ -128,7 +129,7 @@ class TransferAccountListPage extends React.Component {
 export default connect(
   mapStateToProps,
   mapDispatchToProps
-)(TransferAccountListPage);
+)(organizationWrapper(TransferAccountListPage));
```
Comment (tristanhcole) [inline app/client/components/pages/transferAccountListPage.jsx:132]: organizationWrapper needs to be on the page not subcomponent

File: app/client/components/organizationWrapper.jsx [hunk 2]
```diff
@@ -12,9 +12,9 @@ const OrganizationWrapper = Component =>
       let query_params = parseQuery(location.search);
       if (
         query_params["org"] &&
-        this.props.login.organisationId !== query_params["org"]
+        this.props.login.organisationId !== parseInt(query_params["org"])
       ) {
-        let org_id = query_params["org"];
+        let org_id = parseInt(query_params["org"]);
```
Comment (tristanhcole) [inline app/client/components/organizationWrapper.jsx:17]: `organisationId` is defined as a `number` in `auth/types`

https://github.com/teamsempo/SempoBlockchain/blob/efaf2c66df4f0333c2015bbcfd31c320b261d92f/app/client/reducers/auth/types.ts#L11-L13

https://github.com/teamsempo/SempoBlockchain/blob/efaf2c66df4f0333c2015bbcfd31c320b261d92f/app/client/reducers/auth/types.ts#L35

File: app/client/components/pages/singleUserPage.jsx [hunk 3]
```diff
@@ -1,99 +1,101 @@
...
-import LoadingSpinner from '../loadingSpinner.jsx'
-import {LightTheme} from '../theme.js'
-import SingleUserManagement from '../user/SingleUserManagement.tsx';
-
-import { loadUser } from '../../reducers/userReducer'
-import { loadTransferUsages } from '../../reducers/transferUsage/actions'
-import organizationWrapper from '../organizationWrapper'
-
-const mapStateToProps = (state) => {
+import React from "react";
```
Comment (tristanhcole) [inline app/client/components/pages/singleUserPage.jsx:1]: this is all githook formating

File: app/client/sagas/authSagas.js [hunk 4]
```diff
@@ -123,7 +123,10 @@ function* saveOrgId({ payload }) {
     let query_params = parseQuery(window.location.search);
 
     // if query param and payload are matching then just reload to update navbar
-    if (query_params["org"] && payload.organisationId == query_params["org"]) {
+    if (
+      query_params["org"] &&
+      payload.organisationId.toString() === query_params["org"]
```
Comment (enjeyw) [inline app/client/sagas/authSagas.js:128]: higher up u parse the query param to a number and then compare, perhaps do that here too for consistency?

Comment (tristanhcole) [reply app/client/sagas/authSagas.js]: fixed

**Comment by tristanhcole** [comment]
@enjeyw would be good to merge this small fix in before #265

**Pull request merged**